rotkehlchen/chain/manager.py

```diff
@@ -1527,3 +1527,18 @@ def get_loopring_balances(self) -> Dict[Asset, Balance]:
                 aggregated_balances[asset] += balance
 
         return dict(aggregated_balances)
+
+    def get_chain_manager(
+        self,
+        blockchain: SupportedBlockchain,
+    ) -> Any:
+        if blockchain == SupportedBlockchain.ETHEREUM:
```

Comment:
you could do it even easier with a `getattr`on `blockchain.name.lower`. We have made sure the name of the attribute matches.

Reply:
good point :+1: